crates/nu-cli/src/repl.rs

```diff
@@ -257,10 +272,12 @@ pub fn evaluate_repl(
         );
 
         start_time = std::time::Instant::now();
-        line_editor = add_menus(line_editor, engine_reference, stack, config).unwrap_or_else(|e| {
-            report_error_new(engine_state, &e);
-            Reedline::create()
-        });
+        // STACK-REFERENCE 3
```

Comment:
This does not clone the `Arc`, so this should not be marked as a `STACK-REFERENCE`.